index.Rmd

```diff
@@ -28,15 +28,16 @@ This project explores the number and frequency of updates in a project currently
 
 ## Methods
 
-mon amour - can you write the section on how you extracted the data?
+The [dependency manifest file](https://github.com/canada-ca/tracker/blob/003a3d6e0452fb9a248a7d51fd5556bc03dfbcc6/frontend/package.json) from the frontend of Government of Canada's Tracker project was used as a representative example of a modern software project.
 
-On October 22, 2020, the registry for NPMJS...
+Using the following command, package names (both development and production) were extracted from the `package.json` and details for each was retrieved from the npm API.
 
-`
-curl -sL "https://registry.npmjs.org/$pkg" | jq -r --arg  name "$pkg" '.time | to_entries[] | [$name, .key, .value] | @csv'
-`
+```
+for pkg in $(cat package.json | jq '.dependencies * .devDependencies | keys[]' | tr -d \"); do $(curl -sL "https://registry.npmjs.org/$pkg" | jq -r --arg  name "$pkg" '.time | to_entries[] | [$name, .key, .value] | @csv' >> nodejs.csv && sleep 1 ); done
+```
+The command uses [jq](https://stedolan.github.io/jq) to select dependency name from the json and format the API response as CSV.  [curl](https://curl.se/) was used to retrieve data from the API.
```

Comment:
I would include this sentence before the actual line of code - it provides context to understand the code - do we also want to specify that this was done at the command line or is this obvious already?

Reply:
:brain: Probably best to optimize for clarity.
I moved the sentence and specified that this is a bash script and added a link to clarify.

Reply:
Thanks